spec/controllers/application_controller_spec.rb

```diff
@@ -88,6 +93,15 @@ def append_info_to_payload(payload)
       get :not_authorized
     end
 
+    it 'does log exceptions to sentry based on level identified in exception.en.yml' do
```

Comment:
can there be a test for when the level is `'none'`?

Reply:
there kind of already is -- for `breakers_outage` and others the level is 'none' -- those specs were merged in a previous PR. They are not failing, therefore the behavior is as it was.

Reply:
For reference, the following specs were added before these changes because I didn't have confidence that I would breaking something:

https://github.com/department-of-veterans-affairs/vets-api/pull/5200

Reply:
It looks like I misunderstood this test a little.  I thought it was defining an exception to raise,  but now I see it'd describing the exception it expects to see.
I saw those other specs and I'm glad they cover this case.  I've just found tests to be very helpful when there's explicitly one for each behavior.
So of course I'd like more tests, lol, but the one I thought I was suggesting isn't so simple and I withdraw my comment